src/lib/rmt/shared/set_password_dialog.rb

```diff
@@ -44,7 +44,7 @@ def ok_handler
       return
     elsif password.size < @min_password_size
       Yast::UI.SetFocus(Id(:password))
-      Yast::Report.Error(_('Password has to have at least %<size>s characters.') % { size: @min_password_size })
+      Yast::Report.Error(_('Password has to have at least %{size} characters.') % { size: @min_password_size })
```

Comment:
does auto convert of integer to string happen there? I think it was reason for `s` previously.

Reply:
Works as expected:

```console
# irb
irb(main):001:0> "test: %{foo}" % {foo: 42}
=> "test: 42"

```

Reply:
good